docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -80,19 +80,19 @@ After displaying a widget, you can manage the widget application's lifecycle:
    Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
    ```
 
-2. Resume the instance of a widget application using `widget_viewer_evas_resume_widget()`. The parameter of `widget_viewer_evas_resume_widget()` is a widget object created by `widget_viewer_evas_add_widget()`:
+2. Resume the instance of widget application using `widget_viewer_evas_resume_widget()` with the widget object as the parameter.:
```

Comment:
Please add "the" before "widget application" and change "with" to "using".
Please consider the same for line 89 also.